clusters/app.ci/supplemental-ci-images/telco-bastion.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: List
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    namespace: ocp
```

Comment:
Please use `ci` namespace. The generated imagestream is not a part of OCP.
Note that the image in `ci` is open to the public.


```console
$ oc get is -n ci telco-bastion
Error from server (NotFound): imagestreams.image.openshift.io "telco-bastion" not found
```